addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -13,7 +13,7 @@
                                 'o-bordered': chatterTopbar.chatter.hasExternalBorder,
                                 'border-top border-left': chatterTopbar.chatter.composerView and !chatterTopbar.chatter.composerView.composer.isLog and chatterTopbar.chatter.hasExternalBorder,
                             }"
-                            t-att-disabled="chatterTopbar.chatter.isDisabled"
+                            t-att-disabled="chatterTopbar.chatter.isDisabled or !chatterTopbar.chatter.thread.hasWriteAccess"
```

Comment:
Could move `Thread/hasWriteAccess` in the compute of `Chatter/isDisabled`

Reply:
If I remember well this will make issue because `isDisabled` is used in places where the new behavior is not desirable. I think the current PR is a better minimal diff until field names are cleaned up.

Reply:
That's what I thought with the conditional `disabled` on `AttachmentBox` that completely omit the disable state of `Chatter` somehow, but I'm wondering why this is the case.

Also, I see 5 occurrences of `Chatter/isDisabled` in total in this file, but only 4 being changed in this PR. The one missing is the passing of props `FollowerListMenuComponent/isDisabled`. Why is it not disabled when we don't have write access?

Don't get me wrong, I'm not necessarily saying that all uses `Chatter/isDisabled` should necessarily cover `Chatter/hasWriteAccess`. But seeing the amount of `Chatter/isDisabled or !Chatter/hasWriteAccess` here, there's likely a missing opportunity to adjust modelling:
- disjunctive conditional expressions usually hide an underlying concept
- this expression feels like it's mixing **cause** (`hasWriteAccess`) and **consequence** (`isDisabled`);

I think `Chatter/isDisabled` was badly used in some context, notably in `AttachmentBoxComponent` and `FollowerListMenuComponent`. `Chatter/hasWriteAccess` is a much better base to deduce the disable state of part of UI.